src/Spice86/Converters/InstructionToStringConverter.cs

```diff
@@ -0,0 +1,35 @@
+namespace Spice86.Converters;
+
+using Avalonia.Collections;
+using Avalonia.Data.Converters;
+
+using Iced.Intel;
+
+using Spice86.Models.Debugging;
+
+using System.Globalization;
+using System.Text;
+
+public class InstructionToStringConverter : IValueConverter {
+    private readonly StringBuilder _outputString = new();
+    private readonly Formatter _formatter = new MasmFormatter();
+    public object? Convert(object? value, Type targetType, object? parameter, CultureInfo culture) {
+        if (value is CpuInstructionInfo cpuInstructionInfo) {
+            Instruction instr = cpuInstructionInfo.Instruction;
+            _outputString.Clear();
+            _formatter.Options.DigitSeparator = "`";
+            _formatter.Options.FirstOperandCharIndex = 10;
```

Comment:
Can these be moved to the initialization of the formatter, so that they're not set every time?

Reply:
Done.